xsd/siri_estimatedTimetable_service.xsd

```diff
@@ -242,6 +242,16 @@ If false each subscription response will contain the full information as specifi
      <xsd:documentation>The amount of change to the arrival or departure time that can happen before an update is sent (i.e. if ChangeBeforeUpdate is set to 2 minutes, the subscriber will not be told that a timetable is changed by 30 seconds - an update will only be sent when the timetable is changed by at least least 2 minutes. (OPtional from SIRI 2.0)</xsd:documentation>
     </xsd:annotation>
    </xsd:element>
+   <xsd:element name="SkipRecordedCallUpdates" type="xsd:boolean" default="false" minOccurs="0">
+    <xsd:annotation>
+     <xsd:documentation>Indicates whether actual arrival-/departure times should be delivered as incremental updates, i.e. whether RECORDED CALL updates are transmitted immediately after an event occurs. +SIRI v2.1
```

Comment:
It is weird: If Skip is true then we don't Skip Recorded Call updates ... Furthermore, in not Skiping should be the default (for compatility reasons: it was not possible to ask to skip before)

Reply:
Values were switched by mistake. Fixed in newest commit.